app/main.go

```diff
@@ -177,6 +178,9 @@ func (app *SpacemeshApp) before(cmd *cobra.Command, args []string) (err error) {
 	// ensure cli flags are higher priority than config file
 	EnsureCLIFlags(cmd, app.Config)
 
+	// use TimeConfigValues as time config instead of CLI flags
+	app.Config.TIME = timeCfg.TimeConfigValues
```

Comment:
the other way around. `timeCfg.TimeConfigValues` should be set to the values from `app.Config.TIME`

Reply:
we want CLI flags and config file to be loaded. into `timeCfg.TimeConfigValues`. currently they are loaded into `app.Config.TIME`. so just assign `app.Config.TIME` to `timeCfg.TimeConfigValues`.

Reply:
I see. It was a misinterpretation, maybe in the future, it can be changed to pull config in `timesync` from the centralized `app.Config.TIME`

Reply:
`app.Config.Time` is currently only created and loaded in `main`. the only solution I see now would be to change the structure of `timesync` to be created in `main` (it global now) that way it can be used to pass the config down to `timesync` and we will not need `TimeConfigValues`.  it will also mean the created struct instance would need to be passed to all its consumers.
good job 👍